plugins/train/trainer/_base.py

```diff
@@ -25,8 +25,8 @@
 import os
 import time
 
-import cv2
 import numpy as np
+import cv2
```

Comment:
[*super anal mode*] I prefer imports to be in alphabetical order

Reply:
I did it this way because numpy set environment variables to ensure openblas doesnt set the cpu affinity and i wasn't sure if opencv might use openblas too. Doesn't seem like it tho.

Reply:
I did some testing with importing numpy in faceswap.py. I didn't observe any difference.

I will test on Windows before posting my full results.